pkg/kubectlcmd/client.go

```diff
@@ -89,14 +89,38 @@ func (c *Client) Apply(dryRun, verbose bool, kubeconfig, context, namespace stri
 	log.Infof("applying to namespace %s:\n%s\n", namespace, cmdStr)
 
 	err := c.cmdSite.Run(cmd)
+	csError := consolidateError(stderr.String())
+
 	if err != nil {
 		logAndPrint("error running kubectl apply: %s", err)
-		return stdout.String(), stderr.String(), fmt.Errorf("error running kubectl apply: %s", err)
+		return stdout.String(), csError, fmt.Errorf("error running kubectl apply: %s", err)
 	}
 
 	logAndPrint("kubectl apply success")
 
-	return stdout.String(), stderr.String(), nil
+	return stdout.String(), csError, nil
+}
+
+// consolidateError is a helper function to dedup the error messages
```

Comment:
This doesn't have to be error specific, actually it's a useful general function for deduping log lines. I suggest changing the name and vars and putting it in util.

Reply:
ok moved it to util and make it general